ksql-functional-tests/src/test/java/io/confluent/ksql/test/tools/KsqlTestingToolTest.java

```diff
@@ -57,16 +53,13 @@ public void restoreStreams() {
 
   @Test
   public void shouldRunCorrectsTest() throws Exception {
-
-    final File testFolder = new File(CORRECT_TESTS_FOLDER);
-    final File[] testSubFolders = testFolder.listFiles(File::isDirectory);
-
-    for (final File correctTestFolder: testSubFolders) {
```

Comment:
FYI, this is the problematic line. `listFiles` can return `null`, we don't check for it and `spotBugs` complains.